### PR TITLE
Doc(eos_designs): Fix typos in Monitor Session Documentation

### PR DESCRIPTION
**Pull request by ccsnw**
## Change Summary

Pasted old datamodel in documentation. Fixing by adding the actual datamodel that was used

## Related Issue(s)

Fixes #1472 

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes
Correct data model is:

```
monitor_sessions:
  - name: < session_name >
    role: < source | destination >
    source_settings:
      direction: < rx | tx | both >
      access_group:
        type: < ip | ipv6 | mac >
        name: < acl_name >
        priority: < priority >
    session_settings:
      encapsulation_gre_metadata_tx: < true | false >
      header_remove_size: < bytes >
      access_group:
        type: < ip | ipv6 | mac >
        name: < acl_name >
      rate_limit_per_ingress_chip: < "<int> bps" | "<int> kbps" | "<int> mbps" >
      rate_limit_per_egress_chip: < "<int> bps" | "<int> kbps" | "<int> mbps" >
      sample: < int >
      truncate:
        enabled: < true | false >
        size: < bytes >
```

## How to test
Molecule tests

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [o] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_designs/doc/connected-endpoints.md [hunk 1]
```diff
@@ -171,28 +171,28 @@ port_profiles:
...
-        monitor_session:
-          name: < session_name >
-          role: < source | destination >
-          source_settings:
-            - direction: < rx | tx | both >
+        monitor_sessions:
+          - name: < session_name >
+            role: < source | destination >
+            source_settings:
+              - direction: < rx | tx | both >
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_designs/doc/connected-endpoints.md]: this should not be a list either I guess.

Comment (ccsnw) [reply ansible_collections/arista/avd/roles/eos_designs/doc/connected-endpoints.md]: I should stop using computers, obviously. 

:-))

**Review by tgodaA — APPROVED** [review]
LGTM

**Review by ClausHolbechArista — COMMENTED** [review]
I can see that with this data model we will have a failure with the `combine`, if we have multiple sources on the same session. It will overwrite the `source_settings`. So maybe we should take a half step back and parse all sources first, and _then_ do the `combine` and parse the `session_settings`.
Either we could expand this fix, or add another PR with that fix.

**Comment by ccsnw** [comment]
> I can see that with this data model we will have a failure with the `combine`, if we have multiple sources on the same session. It will overwrite the `source_settings`. So maybe we should take a half step back and parse all sources first, and _then_ do the `combine` and parse the `session_settings`. Either we could expand this fix, or add another PR with that fix.

@ClausHolbechArista Are you sure? I'm not using the combined/merged settings to render the sources, only for the dests. In the molecule tests I have the session "MonitoringSessionServer18WithDest" which has multiple sources and different settings on them.

**Comment by ClausHolbechArista** [comment]
> > I can see that with this data model we will have a failure with the `combine`, if we have multiple sources on the same session. It will overwrite the `source_settings`. So maybe we should take a half step back and parse all sources first, and _then_ do the `combine` and parse the `session_settings`. Either we could expand this fix, or add another PR with that fix.
> 
> @ClausHolbechArista Are you sure? I'm not using the combined/merged settings to render the sources, only for the dests. In the molecule tests I have the session "MonitoringSessionServer18WithDest" which has multiple sources and different settings on them.

You are right. It works as expected.

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Pull request merged**